doc/factory/custom_vars.html

```diff
@@ -1709,13 +1709,14 @@ <h3><a name="frontend_vars"></a>Frontend Client Variables</h3>
                 This attribute represents number of jobs completed too quickly
                 before raising the black hole detection flag. More details in
                 black hole detection section, on the bottom of who webpage
-                (Lifetime of a Glidein).
+                (Lifetime of a Glidein). When defined, must be an integer bigger
+                than 0.
```

Comment:
Is this in a new line rather than being a continuation of the previous line?

Reply:
Same for html